Modal/js/script.js

```diff
@@ -0,0 +1,25 @@
+// Get the modal
```

Comment:
commenting is good, but upto a certain extend

commenting what's obvious is not recommented as per clean code practice

Ref:- https://github.com/ryanmcdermott/clean-code-javascript#comments